_rules/aria-presentational-role-no-global-states-properties-p8g918.md

```diff
@@ -0,0 +1,175 @@
+---
+id: p8g918
+name: ARIA presentational role does not have global states or properties
```

Comment:
I'm very sorry to do this a second time. I can tell you've put a lot of work into these, but this rule also seems like a duplicate to me. This time of [Element marked as decorative is not exposed ](https://www.w3.org/WAI/standards-guidelines/act/rules/46ca7f/). Your version makes slightly different trade-offs about explicitly stating which props cause this problem, whereas the existing rule is vaguer about that. But then in being vaguer and just saying "not in the accessibility tree" its means it can handle changes to the ARIA spec like how aria-label is not a global attr if on role=none, but still fails for creating presentation conflict.

Reply:
I see the point. 

The intent was to cover a specific ARIA requirement in detail and comprehensively (Presentational Roles Conflict Resolution) to guarantee authors are clear on what to do in these scenarios. 

I'm uncertain about the next steps. 

The other two existing ARIA requirements are rather more vague but do cover the necessary aspects. If we decide to revert these new rules, we'll still need to add examples to the existing rules.